public/build/css/all.css

```diff
@@ -1,3 +1,7 @@
+/*!
```

Comment:
Yes I was compiling with gulp --production. I think we should minify before commiting.